src/open_inwoner/templates/pages/questions/questions.html

```diff
@@ -0,0 +1,37 @@
+{% load i18n icon_tags list_tags %}
+
+<div class="contactmomenten">
+<h2 class="h3 contactmomenten__title">{% trans "Vragen" %}</h2>
```

Comment:
`<h2>` that has `h3` as its class, not sure if that is intentional?

Reply:
If this is a reusable component, it would be better to only use its own unique classes, like contactmomenten__title, and check if it has the correct heading-font and font-size like in the design.

Reply:
I was following the pattern of `status` heading on the same page, which is also an `h2` with class attribute `h3`. I realize this is strange and I commented on this myself before. However, we ought to be consistent not just with the design but also with our own templates. I thought @jiromaykin was going to take care of `h2` elements as part of a later sprint (our `h1` are already styled with `utrecht-heading`), that's why I wrote it as is.